demo-apps/cf-helloworld-server/src/main/java/org/eclipse/californium/examples/HelloWorldServer.java

```diff
@@ -27,22 +28,30 @@
 import org.eclipse.californium.core.network.EndpointManager;
 import org.eclipse.californium.core.network.config.NetworkConfig;
 import org.eclipse.californium.core.server.resources.CoapExchange;
-
+import org.eclipse.californium.elements.tcp.TcpServerConnector;
 
 public class HelloWorldServer extends CoapServer {
 
-	private static final int COAP_PORT = NetworkConfig.getStandard().getInt(NetworkConfig.Keys.COAP_PORT);
+    private static final int COAP_PORT = NetworkConfig.getStandard().getInt(NetworkConfig.Keys.COAP_PORT);
+    private static final int TCP_THREADS = NetworkConfig.getStandard().getInt(NetworkConfig.Keys.TCP_WORKER_THREADS);
+    private static final int TCP_IDLE_TIMEOUT = NetworkConfig.getStandard()
+            .getInt(NetworkConfig.Keys.TCP_CONNECTION_IDLE_TIMEOUT);
+
     /*
      * Application entry point.
      */
     public static void main(String[] args) {
-        
-        try {
 
+        try {
             // create server
+            boolean udp = true;
+            boolean tcp = false;
+            if (0 < args.length) {
+                tcp = args[0].equalsIgnoreCase("coap+tcp:");
```

Comment:
Please note:
this PR was mainly prepared, because a user had problems to setup the TCP communication. So I thought, it would be a good idea to provide a "working setup".